### PR TITLE
Add worldchain into evms_info, add two more trusted tokens

### PR DESCRIPTION
**Pull request by Hosuke**
### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

File: dbt_subprojects/daily_spellbook/models/evms/evms_info.sql [hunk 1]
```diff
@@ -63,4 +63,5 @@ FROM (
         , (11155111, 'sepolia', 'Sepolia', 'Testnet', 'Testnet', 'tETH', 0x7b79995e5f793a07bc00c21412e50ecae098e7f9, 'https://sepolia.etherscan.io/', timestamp '2015-07-30 15:26:13', NULL, NULL, NULL)
         , (167000, 'taiko', 'Taiko', 'Layer 2', 'Optimistic Rollup', 'ETH', 0xa51894664a773981c6c112c43ce576f315d5b1b6, 'https://taikoscan.network/', timestamp '2024-03-29 07:39:12', 'OP Stack', 'Ethereum Blobs', 'Ethereum')
         , (1329, 'sei', 'Sei Network', 'Layer 1', NULL, 'SEI', 0xE30feDd158A2e3b13e9badaeABaFc5516e95e8C7, 'https://seitrace.com/?chain=pacific-1', timestamp '2024-05-27 13:14', NULL, NULL, NULL)
-) AS temp_table (chain_id, blockchain, name, chain_type, rollup_type, native_token_symbol, wrapped_native_token_address, explorer_link, first_block_time, codebase, data_availability, settlement)
\ No newline at end of file
+        , (480, 'worldchain', 'World Chain', 'Layer 2', NULL, 'WLD', 0x2cFc85d8E48F8EAB294be644d9E25C3030863003, 'https://worldchain-mainnet.explorer.alchemy.com/', timestamp '2024-06-25 17:14', 'OP Stack', 'Ethereum Blobs', 'Ethereum')
```
Comment (Hosuke) [inline dbt_subprojects/daily_spellbook/models/evms/evms_info.sql:66]: References:
https://chainlist.org/chain/480
https://optimism.mirror.xyz/I4OlaW1XCW55FAsS0_S8uUqVz3wLZ-fS-4_f5vuWGEk
https://worldcoin.org/world-chain

Comment (Hosuke) [reply dbt_subprojects/daily_spellbook/models/evms/evms_info.sql:66]: `Optimistic Rollup` Maybe:
```suggestion
        , (480, 'worldchain', 'World Chain', 'Layer 2', 'Optimistic Rollup', 'WLD', 0x2cFc85d8E48F8EAB294be644d9E25C3030863003, 'https://worldchain-mainnet.explorer.alchemy.com/', timestamp '2024-06-25 17:14', 'OP Stack', 'Ethereum Blobs', 'Ethereum')
```

**Review by jeff-dude — APPROVED** [review]
fyi @hildobby

**Comment by jeff-dude** [comment]
looks like no support from our automated erc20 metadata source on this chain (at least for now). may not be a big deal, as that link to show tokens on mainnet tell us the top two dominate, while the other tokens have v few holders

**Comment by jeff-dude** [comment]
@Hosuke if we look at tokens traded on uni v3 for worldchain, it's only 4 total:
```
select distinct token_bought_address
from dex.trades
where blockchain = 'worldchain'
union
select distinct token_sold_address
from dex.trades
where blockchain = 'worldchain'
```

i think for now, we should add these to manual static file. i have followed up with source provider for future coverage

**Comment by Hosuke** [comment]
> @Hosuke if we look at tokens traded on uni v3 for worldchain, it's only 4 total:
> 
> ```
> select distinct token_bought_address
> from dex.trades
> where blockchain = 'worldchain'
> union
> select distinct token_sold_address
> from dex.trades
> where blockchain = 'worldchain'
> ```
> 
> i think for now, we should add these to manual static file. i have followed up with source provider for future coverage

I tracked the tokens here:
https://worldchain-mainnet.explorer.alchemy.com/tokens

Only a few looks farmiliar.

**Pull request merged**